### DISCUSSION TITLE
Will Windows 10 run Theia？

### DISCUSSION THREAD

**Discussion by 7FanZhi** [eclipse-theia/theia#10936] [Q&A]
Will Windows 10 run Theia？

**Comment by msujew** [comment]
Hi @7FanZhi,

yes, you should be able to build and run Theia on Windows 10/11. If you have any trouble setting up a dev environment on Windows, you can take a look at our [guide for building on Windows](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#building-on-windows).
